db/migrate/20180620124108_use_utf8_mb4_for_comments.rb

```diff
@@ -0,0 +1,11 @@
+class UseUtf8Mb4ForComments < ActiveRecord::Migration[5.1]
```

Comment:
What problems may we face if we try to make this change for all tables:
https://gist.github.com/amuntasim/f3b12f20a30e9a9f3fb0